viber-fe/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+import { AppActions } from "../actions";
+import { initialState } from "../store";
+
+function reducer(state = initialState, action) {
+    const { type, payload } = action;
+    switch (type) {
+        case AppActions.LOGIN:
+            return state
+        case AppActions.SIGNUP:
+            return state
+            default:
```

Comment:
Check Indentation

Reply:
I am afk can you do the indent and push?

Reply:
Merge it also.